### ISSUE TITLE
Describe in the WIKI how to use Many Fonts

### ISSUE THREAD

**Issue by ValenDula** [tommyettinger/textratypist#18]
****private val nameFontMedium = "Medium"
    private val nameFontBold   = "Bold"

    **private val fontMediumBold = Font().setFamily(Font.FontFamily(arrayOf(
        Font(fontMedium24).setName(nameFontMedium),
        Font(fontBold24).setName(nameFontBold),
    )))**

    private val textUsed = "[@$nameFontMedium]My Medium Text[@$nameFontBold]My Bold Text"
    
        private val lblUsed       = TypingLabel(textUsed, fontMediumBold, Color.BLACK)

**Comment by tommyettinger** [comment]
Yeah, the docs I have on FontFamily could be better. I usually set the first Font in the family to whatever the default should be, since I think if you reset everything with `[ ]` then it uses the first Font in the family. I also usually call `setFamily()` on the same Font that is first in the family, and use that as my main Font. There's some useful extra tricks you can do, like adding one large set of emoji to a Font, then taking a copy of that original Font and instead adding the game-icons set to it. Those couldn't all fit in one Font, there's just too many glyphs, but they can work in separate Fonts in one FontFamily.

The wiki could really be maintained better, but I kinda am focused on getting in 9+ commits every day (and have been for years), and wiki changes don't count as commits to GitHub's metrics for some reason. I need to update the effects, too (there are some new ones).

I'll see about what I can do to document this and other under-documented things in the repo itself. I might just put .md files in the docs/ folder and copy them into the wiki.
